taxonomy-download_tag.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Author:          Uriahs Victor
+ * Created on:      14/07/2021 (d/m/y)
+ *
+ * @package Neve
+ */
+
+/**
+ * Create pagination for Easy Digital Downloads Archive.
+ */
+function neve_edd_download_nav() {
+
+	global $wp_query;
+
+	$big          = 999999;
+	$search_for   = array( $big, '#038;' );
+	$replace_with = array( '%#%', '&' );
+
+	$pagination = paginate_links(
+		array(
+			'base'      => str_replace( $search_for, $replace_with, get_pagenum_link( $big ) ),
+			'format'    => '?paged=%#%',
+			'current'   => max( 1, get_query_var( 'paged' ) ),
+			'total'     => $wp_query->max_num_pages,
+			'prev_next' => true,
+		) 
+	);
+	?>
+
+	<div id="nv-edd-download-pagination" class="navigation">
+		<?php 
+		//phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+		echo $pagination; 
+		?>
+	</div>
+
+	<?php
+}
```

Comment:
This function gets repeated a lot. Can we have it in a global PHP and just hook it in a `neve_edd_download_nav` hook? 🚀